problemsets-mat223/modules/module11-exercises.tex

```diff
@@ -228,5 +228,42 @@
 					A)=2$ $\Nullity(T) = \Nullity(A)=1$
 			\end{enumerate}
 		\end{solution}
+
+		\prob Give an example of a $3\times4$ matrix $M$ with the specified rank, or explain why one cannot exist.
+		\begin{enumerate}
+			\item $\Rank(M) = 0$
+			\item $\Rank(M) = 1$
+			\item $\Rank(M) = 3$
+			\item $\Rank(M) = 4$
+		\end{enumerate}
+
+		\begin{solution}
+			\begin{enumerate}
+				\item $\mat{0&0&0&0\\0&0&0&0\\0&0&0&0}$
+				\item $\mat{1&0&0&0\\0&0&0&0\\0&0&0&0}$
+				\item $\mat{1&0&0&0\\0&1&0&0\\0&0&1&0}$
+				\item A $3\times4$ matrix with rank 4 cannot exist. Two examples of justifications are: \\
+				1) Thinking about the induced transformation, the 3 dimensional domain cannot produce a 4 dimensional range (Rank-Nullity Theorem).\\
+				2) A matrix with rank 4 has 4 pivots. A $3\times4$ matrix only has 3 rows which makes it impossible to have 4 pivots.
+			\end{enumerate}
+		\end{solution}
+
+		\prob Let $P:\R^3\to\R^3$ be the projection onto the $xy$-plane.
+		\begin{enumerate}
+			\item Find a matrix $M_P$ for the transformation.
+			\item Find the range of $P$.
+			\item Find the column space of $M_P$. Are there any similarities to your answer in the previous part?
+			\item Find the null space of $P$ and $M_P$. Are there similarities between the null space of a linear transformation and its associated matrix?
+		\end{enumerate}
+
+		\begin{solution}
+			\begin{enumerate}
+				\item $\mat{1&0&0\\0&1&0\\0&0&0}$
+				\item the range of $P$ is the $xy$-plane.
+				\item the column space of $M_P$ is $\Span\Set*{\mat{1\\0\\0},\mat{0\\1\\0}} =$ the $xy$-plane. The column space of a matrix is the same as the range of its induced transformation.
+				\item the null space of $P$ and $M_P$ is $\Span\Set*{\mat{0\\0\\1}} =$ the $z$-axis. The null space of a matrix is the same as the null space of its induced transformation.
```

Comment:
Sentences start with capital letters. Also, don't use `= <a word>`. Write it out in English in that case.